backend/src/main/java/com/bakdata/conquery/apiv1/query/TableExportQuery.java

```diff
@@ -135,6 +141,8 @@ public void collectRequiredQueries(Set<ManagedExecutionId> requiredQueries) {
 
 	@Override
 	public void resolve(QueryResolveContext context) {
+		previewConfig = context.getNamespace().getStorage().getPreviewConfig();
```

Comment:
Hier gerne Law of Demeter. Noch greifen wir an vielen stellen direkt auf den Storage zu, aber es wäre schön wenn wir das kapseln in Zukunft